service/controller/kvm/v17/resource/configmap/resource.go

```diff
@@ -18,23 +18,25 @@ type Config struct {
 	ConfigMap configmap.Interface
 	Logger    micrologger.Logger
 
-	CalicoAddress      string
-	CalicoPrefixLength string
-	ClusterIPRange     string
-	ProjectName        string
-	RegistryDomain     string
+	CalicoAddress             string
+	CalicoPrefixLength        string
+	ClusterIPRange            string
+	ControlPlaneWorkerSubnets []string
```

Comment:
At this point when the value has been already parsed from config, would it make sense to have this "properly typed"  as `[]net.IPNet`?

Reply:
I've thought about this, but then again you need to convert it back. But makes sense for validation

Reply:
Added validation for subnets